.changeset/tabaxi-fellas-inspire.md

```diff
@@ -0,0 +1,141 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"__section": tree
+---
+TableSchema (alpha) APIs have been added
+
+Users may now represent dynamic, tabular data using Shared Tree.
+New APIs have been added under a `TableSchema` namespace for creating column, row, and table node schema.
+
+Note: these APIs require the use of [SchemaFactoryAlpha](https://fluidframework.com/docs/api/fluid-framework/schemafactoryalpha-class).
+
+#### Creating a table with default column and row schema
+
+You can craft a table with defaults via `TableSchema.createTable`, without specifying custom column or row schema.
+Note that you will still be required to provide a schema for the cells that will appear in the table.
+
+```typescript
+class Cell extends schemaFactory.object("TableCell", {
+	value: schemaFactory.string,
+}) {}
+
+class Table extends TableSchema.createTable({
+	schemaFactory,
+	cell: Cell,
+}) {}
+
+const table = new Table({
+	columns: [{ id: "column-0" }],
+	rows: [{ id: "row-0", cells: {} }],
```

Comment:
I think this examples should either:

1. Actually, create a cell inline to show the data model for how cells work (put it in the column)

OR

2. Use the methods to specific columns and rows, rather than as inline insertable content. Maybe even have and use a Table.Empty() static to fully hide the details.

Reply:
I like both.

Reply:
Actually, for the latter recommendation, is there any straightforward way for the factory functions to tack on static functionality to the _derived_ schema classes?